### PR TITLE
feat: endow token manager contract

### PR DESCRIPTION
**Pull request by eydbk**
interim commit

### CODE REVIEW

File: contracts/campaigns/CampaignLauncher.sol [hunk 1]
```diff
@@ -210,7 +211,7 @@ contract CampaignLauncher is Ownable {
         maxOpenDonations = _maxOpenDonations;
     }
 
-    function getHash(string memory s) private pure returns (bytes32) {
+    function hash(string memory s) private pure returns (bytes32) {
```
Comment (ttymarucr) [inline contracts/campaigns/CampaignLauncher.sol]: revert this

File: contracts/campaigns/CampaignLauncher.sol [hunk 2]
```diff
@@ -102,7 +103,7 @@ contract CampaignLauncher is Ownable {
     // @todo add reentrancy check
     // user calls the submit to cast newly created vote
     function submit(string calldata _campaignName, uint256 _goal, uint256 _startDate, uint256 _endDate, string memory _cid) external{
-        bytes32 hash = getHash(_campaignName);
+        bytes32 hash = hash(_campaignName);
```
Comment (ttymarucr) [inline contracts/campaigns/CampaignLauncher.sol]: revert this

File: contracts/pools/AAVEDAIDonorsPool.sol [hunk 3]
```diff
@@ -14,70 +14,60 @@ import "../tokens/IToken.sol";
...
     ILendingPool public aaveLendingPool = ILendingPool(0x9198F13B08E299d85E096929fA9781A1E3d5d827);
     ITokenManager private tokenManager = ITokenManager(0xA6687fe9F472159be142b584D3756634C2A1883a);
 
     IToken public token;
-    
+
     mapping(address => uint256) public deposits;
     mapping(address => uint256) public donationPercentages;
-    mapping(uint256=>bool) public donated;
+    mapping(uint256 => bool) public override donated;
```
Comment (ttymarucr) [inline contracts/pools/AAVEDAIDonorsPool.sol:34]: remove override

**Review by ttymarucr — CHANGES_REQUESTED** [review]

File: contracts/pools/AAVEDAIDonorsPool.sol (hunk 3 above)
Comment (eydbk) [reply contracts/pools/AAVEDAIDonorsPool.sol:34]: Notwithstanding the linter generates this error ```Overriding public state variable is missing "override" specifier.``` ? lets see if it compiles

Comment (ttymarucr) [reply contracts/pools/AAVEDAIDonorsPool.sol:34]: ok keep the override

**Review by ttymarucr — APPROVED** [review]

**Pull request merged**